app/assets/javascripts/discourse/app/components/choose-topic.js

```diff
@@ -16,6 +16,7 @@ export default Component.extend({
   label: null,
   loadOnInit: false,
   topicChangedCallback: null,
+  selectIfSingleTopic: null,
```

Comment:
I don't think it needs to be an option. This is a good default to have 👍

Reply:
Made it so.